native-sql-engine/core/src/main/scala/com/intel/oap/expression/ColumnarLiterals.scala

```diff
@@ -140,7 +141,10 @@ class ColumnarLiteral(lit: Literal)
           case null =>
             (TreeBuilder.makeNull(resultType), resultType)
           case _ =>
-            (TreeBuilder.makeLiteral(value.asInstanceOf[java.lang.Long]), resultType)
+            val origLongNode = TreeBuilder.makeLiteral(value.asInstanceOf[java.lang.Long])
+            val timestampNode = TreeBuilder.makeFunction(
+              "seconds_to_timestamp", Lists.newArrayList(origLongNode), resultType)
```

Comment:
To fix this issue: Failed to make LLVM module due to Return type of root node int64 does not match that of expression timestamp[us, tz=UTC]